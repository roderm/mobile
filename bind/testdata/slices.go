package slices

type ByteArray []byte

type S struct {
	B []byte
	b []byte
}

type (s *S) Set(b1 []byte, b2 []byte) {
	s.b = b1
	s.B = b2
}

type (s *S) Get() ([]byte, []byte){
	return s.b, s.B
}