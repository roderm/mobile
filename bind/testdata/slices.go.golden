// Code generated by gobind. DO NOT EDIT.

// Package main is an autogenerated binder stub for package slices.
//
//   autogenerated by gobind -lang=go slices
package main

/*
#include <stdlib.h>
#include <stdint.h>
#include "seq.h"
#include "slices.h"

*/
import "C"

import (
	_seq "golang.org/x/mobile/bind/seq"
	"slices"
)


// suppress the error if seq ends up unused
var _ = _seq.FromRefNum

func proxybasictypes__IntArrays(param_x C.nintslice) C.nintslice {
	_param_x := toSlice(param_x, false)
	res_0 := basictypes.ByteArrays(_param_x)
	_res_0 := fromSlice(res_0, true)
	return _res_0
}
